### ISSUE TITLE
ci instability: test_ui_api -> view_ring_health net::ERR_NETWORK_CHANGED

### ISSUE THREAD

**Issue by jgehrcke** [opstrace/opstrace#1125]
```
[2021-07-27T12:31:54Z]   6 passing (8m)
[2021-07-27T12:31:54Z]   1 failing
[2021-07-27T12:31:54Z]
[2021-07-27T12:31:54Z]   1) test_ui_api
[2021-07-27T12:31:54Z]        view_ring_health:
[2021-07-27T12:31:54Z]      page.goto: net::ERR_NETWORK_CHANGED at https://schedul-bk-2742-497-g.opstracegcp.com/
[2021-07-27T12:31:54Z] =========================== logs ===========================
[2021-07-27T12:31:54Z] navigating to "https://schedul-bk-2742-497-g.opstracegcp.com/", waiting until "load"
```
https://buildkite.com/opstrace/scheduled-main-builds/builds/2742#2623c7b6-e70c-4275-95e3-3e43f3d76203/4821-5524

**Comment by jgehrcke** [comment]
I still don't really know what `net::ERR_NETWORK_CHANGED` is but I suspect that in a browser-based test suite we want to tackle those with a high-level retry. This test is supposed to be in test-browser: https://github.com/opstrace/opstrace/issues/1124 -- there, we retry.

**Comment by jgehrcke** [comment]
```
[2021-07-28T01:21:04Z]
[2021-07-28T01:21:04Z]   6 passing (5m)
[2021-07-28T01:21:04Z]   1 failing
[2021-07-28T01:21:04Z]
[2021-07-28T01:21:04Z]   1) test_ui_api
[2021-07-28T01:21:04Z]        view_ring_health:
[2021-07-28T01:21:04Z]      page.waitForNavigation: Timeout 30000ms exceeded.
[2021-07-28T01:21:04Z] =========================== logs ===========================
[2021-07-28T01:21:04Z] waiting for navigation until "load"
[2021-07-28T01:21:04Z] ============================================================
[2021-07-28T01:21:04Z]       at Context.<anonymous> (test_ui_api.ts:594:16)
```
https://buildkite.com/opstrace/prs/builds/5222#fc5e6d38-261d-4345-91ae-388be78c7a48/1127-1773

**Comment by jgehrcke** [comment]
```
[2021-07-28T09:59:53Z]     1) view_ring_health
[2021-07-28T09:59:53Z] 2021-07-28T09:59:53.820Z info: suite teardown
[2021-07-28T09:59:53Z] 2021-07-28T09:59:53.820Z info: browser.close()
[2021-07-28T09:59:53Z]   pw:api => browser.close started +2ms
[2021-07-28T09:59:53Z]   pw:api <= browser.close succeeded +31ms
[2021-07-28T09:59:53Z] 2021-07-28T09:59:53.852Z info: suite teardown done
[2021-07-28T09:59:53Z]
[2021-07-28T09:59:53Z]
[2021-07-28T09:59:53Z]   6 passing (6m)
[2021-07-28T09:59:53Z]   1 failing
[2021-07-28T09:59:53Z]
[2021-07-28T09:59:53Z]   1) test_ui_api
[2021-07-28T09:59:53Z]        view_ring_health:
[2021-07-28T09:59:53Z]      page.goto: net::ERR_NETWORK_CHANGED at https://pr-upgr-bk-5230-96f-g.opstrace.io/
[2021-07-28T09:59:53Z] =========================== logs ===========================
[2021-07-28T09:59:53Z] navigating to "https://pr-upgr-bk-5230-96f-g.opstrace.io/", waiting until "load"
[2021-07-28T09:59:53Z] ============================================================
[2021-07-28T09:59:53Z]       at Context.<anonymous> (test_ui_api.ts:593:16)
[2021-07-28T09:59:53Z] 
[2021-07-28T09:59:53Z]
```
https://buildkite.com/opstrace/prs/builds/5230#e6906298-8dca-42bd-a36b-4eb742d26f5c/5972-6550

**Comment by jgehrcke** [comment]
let's have this ticket track only `net::ERR_NETWORK_CHANGED`, not the `page.waitForNavigation: Timeout 30000ms exceeded.` one.

**Comment by jgehrcke** [comment]
```
[2021-08-02T16:32:38Z]   6 passing (6m)
[2021-08-02T16:32:38Z]   1 failing
[2021-08-02T16:32:38Z]
[2021-08-02T16:32:38Z]   1) test_ui_api
[2021-08-02T16:32:38Z]        view_ring_health:
[2021-08-02T16:32:38Z]      page.goto: net::ERR_NETWORK_CHANGED at https://prs-bk-5286-aca-g.opstracegcp.com/
[2021-08-02T16:32:38Z] =========================== logs ===========================
[2021-08-02T16:32:38Z] navigating to "https://prs-bk-5286-aca-g.opstracegcp.com/", waiting until "load"
[2021-08-02T16:32:38Z] ============================================================
```
https://buildkite.com/opstrace/prs/builds/5286#bd05d74e-e47b-4209-b5f1-47c3921ec366/4903-5495

**Comment by triclambert** [comment]
```
[2021-08-09T16:48:19Z]   6 passing (6m)
[2021-08-09T16:48:19Z]   1 failing
[2021-08-09T16:48:19Z]
[2021-08-09T16:48:19Z]   1) test_ui_api
[2021-08-09T16:48:19Z]        view_ring_health:
[2021-08-09T16:48:19Z]      page.goto: net::ERR_NETWORK_CHANGED at https://schedul-bk-2825-a64-g.opstracegcp.com/
[2021-08-09T16:48:19Z] =========================== logs ===========================
[2021-08-09T16:48:19Z] navigating to "https://schedul-bk-2825-a64-g.opstracegcp.com/", waiting until "load"
[2021-08-09T16:48:19Z] ============================================================
[2021-08-09T16:48:19Z]       at Context.<anonymous> (test_ui_api.ts:593:16)
``` 

https://buildkite.com/opstrace/scheduled-main-builds/builds/2825#b941ed21-200d-4f13-bd14-f38dc5088bf7/5082-5740

**Comment by jgehrcke** [comment]
We got this before @terrcin landed #1184 
```
[2021-08-10T00:49:59Z]   6 passing (7m)
[2021-08-10T00:49:59Z]   1 failing
[2021-08-10T00:49:59Z]
[2021-08-10T00:49:59Z]   1) test_ui_api
[2021-08-10T00:49:59Z]        view_ring_health:
[2021-08-10T00:49:59Z]
[2021-08-10T00:49:59Z]       AssertionError [ERR_ASSERTION]: page loaded
[2021-08-10T00:49:59Z]       + expected - actual
[2021-08-10T00:49:59Z]
[2021-08-10T00:49:59Z]       -false
```
it's a slightly different symptom. Noting here and not in new ticket because `main` evolved beyond this.
https://buildkite.com/opstrace/scheduled-main-builds/builds/2827#bf7eb519-c40b-4950-88cc-b357fc8bba67/5082-5786
